src/test/base58_tests.cpp

```diff
@@ -82,20 +82,4 @@ BOOST_AUTO_TEST_CASE(base58_DecodeBase58)
     BOOST_CHECK(!DecodeBase58Check("3vQB7B6MrGQZaxCuFg4oh\0" "0IOl"s, result, 100));
```

Comment:
In commit 3b66b7e20ddba94d251c97818700d46030b14cc5

`s/Spit/Split` in commit message.

Reply:
Hah, indeed, proofread all other commit messages:
```bash
git range-diff 836d733a6788c7f7e66da6f8007fb62f15161429..6983d82c84d4ca351a7cd9d1e0e20ab878da6475 ae40cf1a8e16462a8b9dfd076d440bc8ec796c2b..f919d919eb8425ef2bb25aa0ebf61c90ab9b07fa
```